app/views/categories/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="form_container">
+  <div class="navbar">
+  <i class="fas fa-arrow-left"></i>
+  <span class="sr-only">Back</span>
+  <h2>New Category</h2>
+    <%= form_for @category, html: { class: "navbar-form" } do |f| %>
+    <%= link_to categories_path do %>
+  <%= f.submit "Next", class: "btn btn-primary" %>
```

Comment:
:red_circle: [REQUIRED] I'm reviewing your code for the category creation feature. I noticed that the category is not getting saved in the database. The category table is still empty. 

![not showing](https://github.com/danutibe07/BudgetApp/assets/26736582/6313b55f-7a8a-488d-8790-b754906068ee)

 You can verify this by checking the category table in the database.
![not showing](https://github.com/danutibe07/BudgetApp/assets/26736582/60b8cb26-e270-427e-b3f7-1dec03c7138c)